subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/internal/CodeNarcInvoker.groovy

```diff
@@ -47,8 +47,17 @@ abstract class CodeNarcInvoker {
             try {
                 ant.codenarc(ruleSetFiles: "file:${configFile}", maxPriority1Violations: maxPriority1Violations, maxPriority2Violations: maxPriority2Violations, maxPriority3Violations: maxPriority3Violations) {
                     reports.enabled.each { SingleFileReport r ->
-                        report(type: r.name) {
-                            option(name: 'outputFile', value: r.destination)
+                        // See http://codenarc.sourceforge.net/codenarc-TextReportWriter.html
+                        if (r.name == 'console') {
+                            setLifecycleLogLevel(ant,'INFO')
```

Comment:
Following Gradle's code style rules, this should be `(ant, 'INFO')` <- with a _space_